src/mudlet.cpp

```diff
@@ -1936,7 +1936,6 @@ void mudlet::createMapper( bool isToLoadDefaultMapFile )
 
     if( isToLoadDefaultMapFile && pHost->mpMap->mpRoomDB->getRoomIDList().isEmpty() )
     {
-        qDebug() << "mudlet::slot_mapper() - restore map case 3.";
```

Comment:
There are (at least two) other instances of text like this - I needed to track where the map loading was being initiated from for all cases...
Note that the same "case #" actually appears in the `errors.txt` file as well for cross-reference purposes (in the next line of code) so perhaps this one can be let by unmolested at this point.

:thought_balloon:  I peppered the `HostManager` class with `qDebugs()` to do with providing shared-read but exclusive write access to the data (particularly the profile's name) to debug a mutex lock-out issues fairly recently - the text is now actually a bit spammy so could usefully be commented out (rather than deleted ?) I guess.